kong/pdk/log.lua

```diff
@@ -791,72 +804,56 @@ do
           upstream_uri = upstream_uri .. "?" .. (var.args or "")
         end
       end
-
-      return edit_result(ctx, {
+      local root = {
         request = {
           uri = request_uri,
           url = var.scheme .. "://" .. var.host .. ":" .. host_port .. request_uri,
           querystring = okong.request.get_query(), -- parameters, as a table
           method = okong.request.get_method(), -- http method
           headers = okong.request.get_headers(),
           size = request_size,
-          tls = request_tls
+          tls = build_tls_info(var),
         },
         upstream_uri = upstream_uri,
         response = {
           status = ongx.status,
           headers = ongx.resp.get_headers(),
           size = response_size,
         },
-        tries = (ctx.balancer_data or {}).tries,
```

Comment:
why move this line down? any reason?

Reply:
> why move this line down? any reason?

after so, values since this line are same in two serialize function. I'm not sure if any behaviour is based on the order.